packages/neuron-wallet/src/services/transaction-sender.ts

```diff
@@ -928,8 +931,8 @@ export default class TransactionSender {
   public getPrivateKeys = (wallet: Wallet, paths: string[], password: string): PathAndPrivateKey[] => {
     const masterPrivateKey = wallet.loadKeystore().extendedPrivateKey(password)
     const masterKeychain = new Keychain(
-      Buffer.from(masterPrivateKey.privateKey, 'hex'),
-      Buffer.from(masterPrivateKey.chainCode, 'hex')
+      Buffer.from(masterPrivateKey.privateKey.slice(2), 'hex'),
+      Buffer.from(masterPrivateKey.chainCode.slice(2), 'hex')
```

Comment:
```suggestion
      Buffer.from(bytes.bytify(masterPrivateKey.privateKey)),
      Buffer.from(bytes.bytify(masterPrivateKey.chainCode))
```

Reply:
Done. `bytify` is also my good friend now.